app/operations/team_operations/update.rb

```diff
@@ -0,0 +1,34 @@
+module TeamOperations
+  class Update
+    include Operation
+
+    def initialize(team:, team_attrs:, current_user:)
+      @team = team
+      @team_attrs = team_attrs
+      @current_user = current_user
+    end
+
+    def call
+      ActiveRecord::Base.transaction do
+        yield update_story
+
+        Success(team)
+      end
+    rescue
+      Failure(team)
+    end
+
+    private
+
+    attr_reader :team, :team_attrs, :current_user
+
+    def update_story
+      team.attributes = team_attrs
+      if team.save
+        Success(team)
+      else
+        Failure(team)
+      end
```

Comment:
For this case, I think we can do this logic directly inside the `call` method, and we don't need a transaction for that since we're touching a single record.

Reply:
done